packages/gcp/data_stream/firestore/manifest.yml

```
@@ -0,0 +1,31 @@
+title: "GCP Firestore Metrics"
+type: metrics
+streams:
+  - input: gcp/metrics
+    title: GCP Firestore Metrics
+    description: Collect GCP Firestore Metrics
+    vars:
+      - name: zone
+        type: text
+        title: GCP Zone
+        multi: false
+        required: false
+        show_user: true
+      - name: region
+        type: text
+        title: GCP Region
+        multi: false
+        required: false
+        show_user: true
+      - name: period
+        type: text
+        title: Period
+        default: 10s
```

Instead of fixing the period in the test case, let's change the default here, as `10s` is wrong and triggers an error in metricbeat.

```suggestion
        default: 60s
```